en/flight_controller/pixhawk4_mini.md

```diff
@@ -0,0 +1,113 @@
+  * 8 PWM outputs
+  * 4 dedicated PWM/Capture inputs on FMU
+  * Dedicated R/C input for CPPM
+  * Dedicated R/C input for Spektrum / DSM and S.Bus with analog / PWM RSSI input
+  * 3 general purpose serial ports
+  * 2 I2C ports
+  * 3 SPI buses
+  * 1 CANBuses for CAN ESC
+  * Analog inputs for voltage / current of battery
+  * 2 additional analog input
+* Power System:
+  * USB Power Input: 4.75~5.25V
+  * Servo Rail Input: 0~36V
+  * Max current sensing: 120A
+  * Max input voltage: 6V
+* Weight and Dimensions:
+  * Dimensions: 38x55x15.5mm
+* Other Characteristics:
+  * Operating temperature: -40 ~ 85°c
+
+Additional information can be found in the [*Pixhawk 4 Mini* Technical Data Sheet](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/pixhawk4mini/pixhawk4mini_technical_data_sheet.pdf).
+
+## Purchase
+Available soon.
+<!-- Order from [Holybro](https://shop.holybro.com/). -->
+
+## Interfaces
+
+![Pixhawk 4 Mini interfaces](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_interfaces.png)
+
+> **Warning** The **RC IN** and **PPM** ports are for RC receivers only. These are powered! NEVER connect any servos, power supplies or batteries (or to any connected receiver).
+
+
+## Pinouts
+
+Download *Pixhawk 4 Mini* pinouts from [here](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/pixhawk4mini/pixhawk4mini_pinouts.pdf).
+
+## Dimensions
+
+![Pixhawk 4 Dimensions](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_dimensions.png)
+
+## Voltage Ratings
+
+*Pixhawk 4 Mini* can be redundant on the power supply if two power sources are supplied. The power rails are: **POWER** and **USB**.
+
+> **Note** The output power rail **MAIN OUT** does not power the flight controller board (and is not powered by it). You must supply power to one of **POWER** or **USB** or the board will be unpowered.
+
+**Normal Operation Maximum Ratings**
+
+Under these conditions all power sources will be used in this order to power the system:
+1. **POWER** (4.9V to 5.5V)
+1. **USB** input (4.75V to 5.25V)
+
+**Absolute Maximum Ratings**
+
+Under these conditions the system will not draw any power (will not be operational), but will remain intact.
+1. **POWER** input (operational range 4.1V to 5.7V, 0V to 10V undamaged)
+1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
+1. Servo input: VDD_SERVO pin of **MAIN OUT** (0V to 36V undamaged)
+
+
+## Assembly/Setup
+
+The [*Pixhawk 4 Mini* Wiring Quick Start](../assembly/quick_start_pixhawk4_mini.md) provides instructions on how to assemble required/important peripherals including GPS, Power Management Board, etc.
+
+
+## Building Firmware
+
+`make px4fmu-v5_default upload`
+
+
+## Debug Port
+
+The system's serial console and SWD interface runs on the **FMU Debug** port, while the I/O console and SWD interface can be accessed via **I/O Debug** port. In order to access these ports, the user must remove the *Pixhawk 4 Mini* casing.
```

Comment:
Are these ports clearly labeled? If not, can we have a screenshot with labels here?

Reply:
I actually havent seen an image of this. Ill ask jinger to get me one.